standards/application/waku-api.md

```diff
@@ -107,90 +107,152 @@ language_mappings:
+        constraints: ["client", "server"]
+        description: "The mode of operation of the Waku node.
+                      Waku core protocols used by the node are inferred from this mode."
       network_config:
         type: struct
-        default: TheWakuNetworkPreset
         fields:
-          boostrap_nodes:
+          boostrap_servers:
             type: array<string>
-            default: ""
-            description: "Bootstrap nodes, entree and multiaddr formats are accepted."
-          static_store_nodes:
+            default: "enrtree://AIRVQ5DDA4FFWLRBCHJWUWOO6X6S4ZTZ5B667LQ6AJU6PEYDLRD5O@sandbox.waku.nodes.status.im"
+            description: "Bootstrap nodes. entree, ENRs list formats are accepted.
+                          This allows the node to discover server nodes."
+            examples:
+              ex1: "enrtree://AIRVQ5DDA4FFWLRBCHJWUWOO6X6S4ZTZ5B667LQ6AJU6PEYDLRD5O@sandbox.waku.nodes.status.im"
+              ex2: ["enr:-QESuED0qW1BCmF-oH_ARGPr97Nv767bl_43uoy70vrbah3EaCAdK3Q0iRQ6wkSTTpdrg_dU_NC2ydO8leSlRpBX4pxiAYJpZIJ2NIJpcIRA4VDAim11bHRpYWRkcnO4XAArNiZub2RlLTAxLmRvLWFtczMud2FrdS5zYW5kYm94LnN0YXR1cy5pbQZ2XwAtNiZub2RlLTAxLmRvLWFtczMud2FrdS5zYW5kYm94LnN0YXR1cy5pbQYfQN4DgnJzkwABCAAAAAEAAgADAAQABQAGAAeJc2VjcDI1NmsxoQOTd-h5owwj-cx7xrmbvQKU8CV3Fomfdvcv1MBc-67T5oN0Y3CCdl-DdWRwgiMohXdha3UyDw","enr:-QEkuED9X80QF_jcN9gA2ZRhhmwVEeJnsg_Hyg7IFCTYnZD0BDI7a8HArE61NhJZFwygpHCWkgwSt2vqiABXkBxzIqZBAYJpZIJ2NIJpcIQiQlleim11bHRpYWRkcnO4bgA0Ni9ub2RlLTAxLmdjLXVzLWNlbnRyYWwxLWEud2FrdS5zYW5kYm94LnN0YXR1cy5pbQZ2XwA2Ni9ub2RlLTAxLmdjLXVzLWNlbnRyYWwxLWEud2FrdS5zYW5kYm94LnN0YXR1cy5pbQYfQN4DgnJzkwABCAAAAAEAAgADAAQABQAGAAeJc2VjcDI1NmsxoQPFAS8zz2cg1QQhxMaK8CzkGQ5wdHvPJcrgLzJGOiHpwYN0Y3CCdl-DdWRwgiMohXdha3UyDw"]
+          static_servers:
             type: array<string>
             default: []
-            description: "Only the passed nodes are used for store queries, discovered store nodes are discarded."
+            description: "Array of server nodes' multiaddresses. Discovered nodes may also be considered."
+            examples:
+              ex1: ["/dns4/node-01.do-ams3.waku.sandbox.status.im/tcp/30303/p2p/16Uiu2HAmNaeL4p3WEYzC9mgXBmBWSgWjPHRvatZTXnp8Jgv3iKsb"]
+              ex2: ["/dns4/node-01.do-ams3.waku.sandbox.status.im/tcp/8000/wss/p2p/16Uiu2HAmNaeL4p3WEYzC9mgXBmBWSgWjPHRvatZTXnp8Jgv3iKsb"]
           clusterId:
             type: uint
+            constraints: mode == "server"
             default: 1
-          sharding_mode:
-            constraints: ["auto", "static"]
-          auto_sharding_config:
-            type: optionAutoShardingConfig
-            default: none
-            description: "The auto-sharding config, if sharding mode is `auto`"
-          active_relay_shards:
+            description: "The cluster subscribed to and participating in. A cluster is composed by multiple shards."
+          shards:
```

Comment:
You seem to have ignored the original PR. The concept of `shards` in a configuration is very ambiguous. Are we talkin about shards available in the cluster? are we talking about shards for relay to subscribe to?